Services/Saml/classes/class.ilSamlSettingsGUI.php

```diff
@@ -118,6 +122,7 @@ public function __construct($ref_id)
 		$this->tpl           = $DIC->ui()->mainTemplate();
 		$this->lng           = $DIC->language();
 		$this->access        = $DIC->access();
+		$this->rbacsystem    = $DIC->rbac();
```

Comment:
The member variable should be named `rbac` if you don't assign an instance of `\ilRbacSystem`.